pkgs/development/compilers/crystal/default.nix

```
@@ -27,6 +27,8 @@
   tzdata,
   which,
   zlib,
+  # Configuration
+  withRepl ? stdenv.hostPlatform.isx86_64
```

```suggestion
  withRepl ? stdenv.hostPlatform.isx86_64,
```

treefmt is failing https://github.com/NixOS/nixpkgs/actions/runs/15839197758/job/44648675296?pr=380842